rinja_derive/src/lib.rs

```diff
@@ -244,28 +245,13 @@ struct CompileError {
 }
 
 impl CompileError {
-    fn new<S: fmt::Display>(msg: S, file_info: Option<FileInfo<'_>>) -> Self {
-        Self::new_with_span(msg, file_info, None)
-    }
-
-    fn new_with_span<S: fmt::Display>(
-        msg: S,
-        file_info: Option<FileInfo<'_>>,
-        span: Option<Span>,
-    ) -> Self {
+    fn new<S: fmt::Display>(msg: S, file_info: Option<FileInfo<'_>>, span: Option<Span>) -> Self {
         let msg = match file_info {
             Some(file_info) => format!("{msg}{file_info}"),
             None => msg.to_string(),
         };
         Self { msg, span }
     }
-
-    fn no_file_info<S: ToString>(msg: S, span: Option<Span>) -> Self {
```

Comment:
I'm a bit surprised by this removal. I found it more explicit (and easier to `grep`) than having to know what each argument is about...

Reply:
Why would you need to grep this function? Because IDEs tell you the argument name, I think having multiple functions is seldom needed, esp. if it's only so few. And because it's not JavaScript or Python, strong typing comes to your rescue if you missorder the arguments. :)

Reply:
In case I want to know which functions still don't have file information for their error. Also, I don't use an IDE so I don't have this information. ^^'

Reply:
I un-picked the commit.